### DISCUSSION TITLE
Adding compound to aqueous electrolyte property package

### DISCUSSION THREAD

**Discussion by Mathias-olesen** [reaktoro/reaktoro#295] [Q&A]
### Description

Hi,

I am a student working on a carbon capture project in DWSIM, where I have to simulate a small pilot plant using a solution of 30 wt% MEA and 70 wt% water. The problem I am facing is that some of the compounds I can specify in DWSIM are missing in the "Reaktoro Aqueous Electrolytes Property Package" (found here: https://dwsim.org/wiki/index.php?title=Aqueous_Electrolytes_Property_Package). All of the compounds that I need for the simulations are found in the "ChemSep" and "Electrolytes" databases. However, the error I get in DWSIM is that "monoethanolamine" is not supported by the property package. I assume some of the other compounds in the system might also be missing from the property package. The reactions in the system are given in the attached photo.

My question is then: how do I add these compounds to the property package?
Any help is greatly appreciated!

Kind regards, Mathias

![Reactions](https://user-images.githubusercontent.com/99176670/235351372-ef4eb9d5-b8f3-4957-8dc9-58451df76f8e.JPG)

### What is your operating system?

Windows

**Comment by allanleal** [comment]
Hi Mathias,

I'm not sure how this would work out from a DWSIM point of view. The solution from a Reaktoro standpoint depends whether DWSIM is using Reaktoro v1 or v2. Reaktoro v2 has significantly improved the way thermodynamic databases are organized. 

We would need to extend a thermodynamic database of interest supported by Reaktoro ([see here](https://reaktoro.org/tutorials/basics/loading-databases.html)) with the species that you want to use.

[This tutorial](https://reaktoro.org/tutorials/advanced/creating-custom-databases.html) contains information about how databases can be created from scratch or modified. 

Hope this clarifies. Let me know if you need more information.

Best,
Allan

**Reply by Mathias-olesen** [reply]
Hi,

Thank you for the response! I am also in contact with a DWSIM developer, and they say that DWSIM uses Reaktoro v1. However, they also state that Reaktoro might not support the use of amines. Is this correct? 

Kind regards, Mathias

**Comment by allanleal** [comment]
It's possible to extend thermodynamic databases in Reaktoro with additional
species, even amines. The link I shared above explains the process in
general. A collaborator is using Reaktoro v2 for carbon capture modeling
with amines. Please email me for more details.

**Reply by Mathias-olesen** [reply]
Thank you, I will take a look at it. I have also sent you an email regarding your collaborator.

Kind regards, Mathias

**Reply by jprakhar105** [reply]
Hey Mathias, were you able to figure out a solution for this? I am also working on something similar and have hit a roadblock because of this.
